common/shlibs

```diff
@@ -1750,7 +1750,9 @@ libid3.so id3lib-3.8.3_7
 libid3-3.8.so.3 id3lib-3.8.3_7
 libgirara-gtk3.so.4 girara-0.4.4_1
 libjq.so.1 jq-1.6_2
+libcrypto.so.1.1 libcrypto1.1-1.1.1w_1
 libcrypto.so.3 libcrypto3-3.1.2_1
+libssl.so.1.1 libssl1.1-1.1.1w_1
```

Comment:
there should not be any packages linking against libcrypto/libssl 1.1, those only exist for users to use for compat.